src/api/index.js

```diff
@@ -107,8 +107,19 @@ const defineAPI = function(config: IConfig, storage: IStorageHandler) {
 export default (async function(configHash: any) {
   setup(configHash.logs);
   const config: IConfig = new AppConfig(_.cloneDeep(configHash));
+  // register middleware plugins
+  const plugin_params = {
+    config: config,
+    logger: logger,
+  };
+  const filters = loadPlugin(
+    config,
+    config.filters,
```

Comment:
Meh - I used filters here and `storage_filters` in the types. Which do you prefer @juanpicado

Reply:
I went with filters